### DISCUSSION TITLE
How to copy text from an element?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1079] [Q&A] [answered]
I'd like to change keybinding (`ya`) for a HN post. I want to use `ya` to copy the actual link and title instead of the current page. For instance, [this page](https://news.ycombinator.com/item?id=39075304) has a title of `Potential Microbiome Links to Skin Aging | Hacker News`. I want to copy the actual text and linked combined together. Here's the relevant HTML:
```
<td class="title">
  <span class="titleline">
    <a href="https://today.ucsd.edu/story/researchers-discover-potential-microbiome-links-to-skin-aging">Potential Microbiome Links to Skin Aging</a>
  </span>
</td>
```

So I want to copy `Potential Microbiome Links to Skin Aging - https://today.ucsd.edu/story/researchers-discover-potential-microbiome-links-to-skin-aging`

**Comment by gdh1995** [comment, marked as answer]
```
map xxx LinkHints.activateCopyLinkText access=["${innerText} - ${href}"] match="a"
```
